src/agent/debugger/src/debugger.rs

```diff
@@ -205,15 +201,15 @@ pub trait DebugEventHandler {
         // Continue normal exception handling processing
         DBG_EXCEPTION_NOT_HANDLED
     }
-    fn on_create_thread(&mut self, _debugger: &mut Debugger, _info: &CREATE_THREAD_DEBUG_INFO) {}
-    fn on_create_process(&mut self, _debugger: &mut Debugger, _info: &CREATE_PROCESS_DEBUG_INFO) {}
-    fn on_exit_thread(&mut self, _debugger: &mut Debugger, _info: &EXIT_THREAD_DEBUG_INFO) {}
-    fn on_exit_process(&mut self, _debugger: &mut Debugger, _info: &EXIT_PROCESS_DEBUG_INFO) {}
-    fn on_load_dll(&mut self, _debugger: &mut Debugger, _info: &LOAD_DLL_DEBUG_INFO) {}
-    fn on_unload_dll(&mut self, _debugger: &mut Debugger, _info: &UNLOAD_DLL_DEBUG_INFO) {}
+    fn on_create_process(&mut self, _debugger: &mut Debugger, _name: &Path, _base_address: u64 ) {}
+    fn on_create_thread(&mut self, _debugger: &mut Debugger) {}
+    fn on_exit_process(&mut self, _debugger: &mut Debugger, _exit_code: u32) {}
+    fn on_exit_thread(&mut self, _debugger: &mut Debugger, _exit_code: u32) {}
+    fn on_load_dll(&mut self, _debugger: &mut Debugger, _name: &Path, _base_address: u64) {}
```

Comment:
Good idea since the module name might at some point be derived from other parts such as the base address in the uncommon case of loading multiple dlls with the same base name.